staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -33,12 +33,20 @@ var logFlushFreq = pflag.Duration(logFlushFreqFlagName, 5*time.Second, "Maximum
 
 func init() {
 	klog.InitFlags(flag.CommandLine)
+	initLogRegistry()
```

Comment:
@mtaufen is this intended way to add new flags?

Reply:
Why not just call your `options.AddFlags` in the below `AddFlags` function, so they get registered on whatever flagset the consumer is intending to use?

Reply:
AddFlags only called by AddGlobalFlags, and AddGlobalFlags only called in test case, so may be not global implemeted.

Reply:
> AddFlags only called by AddGlobalFlags, and AddGlobalFlags only called in test case, so may be not global implemeted.

@mtaufen